Ginger/Ginger/Environments/AppsListPage.xaml.cs

```diff
@@ -142,5 +142,10 @@ private void AddAppsToOtherEnvironments(object sender, RoutedEventArgs e)
         }
 
         #endregion Functions
+
+        private void EnvNameTextBox_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            grdApps.Title = "'" + EnvNameTextBox.Text + "' Environment Applications";
```

Comment:
Please use String interpolation instead of concatenating string with +

Reply:
Handled.